dts/bindings/led/issi,is31fl3733.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 Daniel DeGrasse <daniel@degrasse.com>
+# SPDX-License-Identifier: Apache-2.0
+description: ISSI IS31FL3733 LED Matrix Driver
+
+compatible: "issi,is31fl3733"
+
+include: "i2c-device.yaml"
+
+
+properties:
+  sdb-gpios:
+    type: phandle-array
+    description: |
+      Hardware shutdown pin. If routed on the board, this property must be
+      present. Set to a logical 1 at boot to exit the device from hardware
+      shutdown.
+  current-limit:
```

Comment:
Please add nl after the first property and so on.

Reply:
Done